test/studies/isx/isx-hand-optimized.chpl

```diff
@@ -132,6 +132,12 @@ config const recvBuffFactor = 2.0,
 config const numBurnInRuns = 1,
              numTrials = 1;
 
+// Horrible hack to help https://github.com/chapel-lang/chapel/issues/9414
+record TimerArr {
+  var A: [1..numTrials] real;
+  proc this(i) ref { return A[i]; }
+  iter these() ref { for a in A do yield a; }
```

Comment:
@mppf I tried to use `forwarding A` here instead of manually forwading this/these, but I got an "already in AST" internal error. I'm having an annoyingly hard time coming up with a smaller reproducer. If you get a second would you mind taking a look and if you don't see anything obvious, just punt it back to me and I'll work more on a smaller reproducer.

Reply:
hey @ronawho, I'm trying to fix it in PR #9456.